docs/vendor/docs-page-shell/page-shell-styles.css

```diff
@@ -0,0 +1,733 @@
+/* ========================================================================== */
+/* shell-prefixed mapbox-assembly + assembly v0.16.0 */
+/* ========================================================================== */
+
+.shell-wrapper ul,
```

Comment:
holly sh*t.... this css is huge.
We got to use a library that comes with a pre-built css...